From d7d329dd9a5dc6e0c6670729f20508f923926e25 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:13 -0400
Subject: [redhat] kabi: Adding symbol seq_release (fs/seq_file.c)

Message-id: <8cf1457ffea798f8f9c10bbbe5cd075c39de9568.1410510773.git.poros@redhat.com>
Patchwork-id: 96008
O-Subject: [PATCH RHEL7.1 193/233] kabi: Adding symbol seq_release (fs/seq_file.c)
Bugzilla: 1071099
RH-Acked-by: Alexander Viro <aviro@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Free the structures associated with sequential file.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/seq_release
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/seq_release
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/seq_release

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/seq_release b/redhat/kabi/kabi-module/kabi_ppc64/seq_release
new file mode 100644
index 0000000..831e67a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/seq_release
@@ -0,0 +1,2 @@
+#1-
+0x58bbe1e6 seq_release vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/seq_release b/redhat/kabi/kabi-module/kabi_s390x/seq_release
new file mode 100644
index 0000000..a83f9ac
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/seq_release
@@ -0,0 +1,2 @@
+#1-
+0x147670c4 seq_release vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/seq_release b/redhat/kabi/kabi-module/kabi_x86_64/seq_release
new file mode 100644
index 0000000..37da3d6
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/seq_release
@@ -0,0 +1,2 @@
+#1-
+0x4f897611 seq_release vmlinux EXPORT_SYMBOL
-- 
1.7.1